extensions/amp-video/0.1/test/test-amp-video.js

```diff
@@ -163,15 +169,11 @@ describes.realWin(
       const video = v.querySelector('video');
       expect(video.getAttribute('src')).to.be.null;
       const sources = video.querySelectorAll('source');
-      expect(sources.length).to.equal(2);
-      expect(sources[0].getAttribute('src')).to.equal(
-        'https://example-com.cdn.ampproject.org/m/s/video.mp4'
-      );
-      expect(sources[1].getAttribute('src')).to.equal(
-        'https://example.com/video.mp4'
-      );
-      expect(sources[0].getAttribute('data-bitrate')).to.equal('1000');
-      expect(sources[1].getAttribute('data-bitrate')).to.equal('1000');
+      expect(sources.length).to.equal(4);
+      expect(sources[0].getAttribute('data-bitrate')).to.equal('2000');
+      expect(sources[1].getAttribute('data-bitrate')).to.equal('720');
+      expect(sources[2].getAttribute('data-bitrate')).to.equal('400');
+      expect(sources[3].hasAttribute('data-bitrate')).to.be.false;
```

Comment:
If the source comes with a bitrate should we list that bitrate here or remove it and rely on the other qualities?

Reply:
Good Q.
If the publisher provided a bitrate themselves, we should keep it. One implementation could be:
- When creating the last source with `amp_quality=...`, if the publisher provided a bitrate themselves, move `data-bitrate` to `data-amp-orig-bitrate`
- Line 502 where you removed the bitrate attr propagation, reintroduce the code but propagate the `data-amp-orig-bitrate`

wdyt?